javascript/javascript3/week3/screenshot app/app.js

```diff
@@ -0,0 +1,114 @@
+console.log('working')
+const BASE_URL = 'https://crudcrud.com/api/'
+const userInput = document.querySelector('#userInput');
+const getBtn = document.querySelector('#getBtn');
+const saveBtn = document.querySelector('#saveBtn');
+const imgContainer = document.querySelector('#imgContainer');
+const showBtn = document.querySelector('#showBtn');
+const savedImgs = document.querySelector('#savedImgs');
+let screenshotURL
+const getScreenshotURL = async() => {
+    if (userInput.value === '') {
+        alert('Please enter url')
+    } else {
```

Comment:
Since you are basically wanting to cancel the rest of the function if the input value is empty, you could instead write this using the [early return pattern](https://gomakethings.com/the-early-return-pattern-in-javascript/), which would make it a little easier to read by not having things be so nested.

ie:
```
const getScreenshotURL = async() => {
    if (userInput.value === '') {
        alert('Please enter url');
	return;
    }
    // ...Rest of the function code    
```

Reply:
There's also a few more places in your code you could use this pattern.